src/components/Tree/Node.tsx

```diff
@@ -173,15 +174,16 @@ export const TreeNode = ({
                     className="tw-p-0 tw-m-0 tw-font-sans tw-font-normal tw-list-none tw-text-left"
                     data-test-id="sub-tree"
                 >
-                    {renderNodeArray(nodes, activeNodeId, onClick, onDrop, [...parentIds, id])}
+                    {renderNodeArray(nodes, activeNodeId, listId, onClick, onDrop, [...parentIds, id])}
```

Comment:
args for renderNodeArray are out of control - we should make it more compact

Reply:
Done

Reply:
Done